build-system/tasks/extension-generator/doc-template.md

```diff
@@ -0,0 +1,76 @@
+<!--
+  1. Change "category" below to one of:
+       ads-analytics
+       dynamic-content
+       layout
+       media
+       presentation
+       social
+  2. Remove any of the "formats" that don't apply.
+     You can also add the "ads" and "stories" formats if they apply.
+  3. And remove this comment! (no empty lines before "---")
+-->
+---
+$category: presentation
```

Comment:
Instead of `presentation`, should this be a placeholder string?

Reply:
After this is merged I'll be updating the template again - lots of feedback is super welcome on that edit if you don't mind waiting :)